### PR TITLE
Files-Update

### PR DESCRIPTION
**Pull request by sanbit876**

### CODE REVIEW

File: yarn.lock
Comment (photon0205) [inline yarn.lock:1]: remove this `yarn.lock` file

**Review by photon0205 — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (sanbit876) [reply yarn.lock:1]: Done

Comment (photon0205) [reply yarn.lock:1]: Do one thing, download the yarn.lock file from the repo and copy and paste it and after this commit it in, actually I meant to untrack the yarn.lock file and not to completely delete it.

Comment (sanbit876) [reply yarn.lock:1]: Made the required changes....please review it

**Review by photon0205 — APPROVED** [review]

**Review by Apoorvgarg-creator — APPROVED** [review]

**Pull request merged**